internal/server/events.go

```diff
@@ -0,0 +1,55 @@
+package server
+
+import (
+	"net/http"
+	"time"
+
+	ihttp "github.com/Pigmice2733/peregrine-backend/internal/http"
+)
+
+type event struct {
+	Name      string    `json:"name"`
+	District  *string   `json:"district,omitempty"`
+	Week      *int      `json:"week,omitempty"`
+	StartDate time.Time `json:"startDate"`
+	EndDate   time.Time `json:"endDate"`
+	Location  struct {
+		Lat float64 `json:"lat"`
+		Lon float64 `json:"lon"`
+	} `json:"location"`
+}
+
+// eventsHandler returns a handler to get all events in a given year
+func (s *Server) eventsHandler() http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		fullEvents, err := s.tba.GetEvents(s.year)
```

Comment:
This is gonna need to get this info from the database. Is this temporary?

Reply:
Yep, will definitely need a database soon.

Reply:
@fharding1 Can this get merged _without_ the database, to unblock me? Then the db can be added in another PR

Reply:
Can't you just write a mock server to respond with a static JSON payload? Or I can do that for you. I'd prefer to not to merge PR's for functionality that we don't actually want.